transactions/common/schema/v3.0/transaction.yaml

```diff
@@ -173,42 +173,18 @@ definitions:
     required:
       - code
       - state
-      - input
-      - output
+      - inputs
+      - outputs
     properties:
       code:
-        anyOf:
-          - type: object
-            additionalProperties: false
-            required:
-              - type
-              - raw
-              - parameters
-            properties:
-              type:
-                type: string
-                enum:
-                  - zenroom
-              raw:
-                type: string
-              parameters:
-                type: array
-                items:
-                  type: object
-          - type: object
-            additionalProperties: false
-            required:
-              - transaction_id
-            properties:
-              transaction_id:
-                "$ref": "#/definitions/sha3_hexdigest"
+        type: string
       state:
         anyOf:
           - type: object
             "$ref": "#/definitions/sha3_hexdigest"
-      input:
+      inputs:
         type: object
-      output:
+      outputs:
         anyOf:
           - type: object
           - type: array
```

Comment:
The definition of PRP10 https://github.com/planetmint/PRPs/tree/main/10#specification
says that inputs and outputs are of type object. I do see the benefits of arrays. But couldn't these be covered by just more complex objects?
If we have both options for outputs, doesn't it also make sense to have that for the inputs?